CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ All notable changes to this project will be documented in this file.
 
 ## [0.2.2] - 2023-03-31
 
+## Breaking Change
+
+Okta JSONNet mapper configuration for SSO login got change a bit. This may impact the Okta user logins. As a workaround use [pinned Okta mapper URL](https://raw.githubusercontent.com/paralus/paralus/v0.2.1/_kratos/oidc-mappers/okta.jsonnet) to your existing Okta OIdC configuration.
```

Comment:
Instead of got changed a bit, it's better to mention the change

Reply:
Fixed.